actiontext/lib/action_text/railtie.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "action_text"
+
+module ActionText
+  class Railtie < Rails::Railtie # :nodoc:
+    config.action_text = ActiveSupport::OrderedOptions.new
+    config.action_text.attachment_tag_name = "action-text-attachment"
+
+    initializer "action_text.configure" do |app|
+      ActionText::Attachment.tag_name = app.config.action_text.attachment_tag_name
+    end
+  end
+end
```

Comment:
Just noticed that this is a new railtie, separate from the existing `ActionText::Engine`. Can we fold this into `ActionText::Engine`, or was this intentional?

Reply:
I just tried to follow patterns looking at other parts of the codebase. I can fold it into `ActionText::Engine`!